lodash.src.js

```diff
@@ -7499,6 +7499,35 @@
     }
 
     /**
+     * Creates a function accepting an array as argument, that invokes `func`
+     * with arguments taken from this array, like `Function#apply`.
+     * This works obviously well with `Promise.all`.
+     *
+     * @static
+     * @memberOf _
+     * @category Function
+     * @param {Function} The function to alter.
+     * @param {*} [thisArg] The `this` binding of `func`.
+     * @returns {*} Returns the new function.
+     * @example
+     *
+     * var spread = _.spread(function (who, what) {
+     *   return who + ' says ' + what;
+     * });
+     *
+     * spread(['John', 'hello']) // => 'John says hello'
+     */
+    function spread (func, thisArg) {
+      if (!isFunction(func)) {
```

Comment:
Also can you make this `typeof func != 'function'`. It's be consistent with some local commits I've done.